style.css

```diff
@@ -508,4 +508,63 @@ main > div img {
  div.evidon-notice-link {
    display:none;
  }
- 
\ No newline at end of file
+ 
+  /*
+  * Styles for Dropdown Region Selector
+  */
+.region-dropdown {
+  border: 1px solid #d3d3d3;
+  background-color: #ffffff;
+  border-radius: 4px;
+  display: inline-block;
+  font-size: inherit;
+  padding: 3px 15px 5px 8px;
+  margin: 0;
+  visibility: hidden;
+  position: absolute;
+  z-index: 0;
+}
+
+.region-dropdown.visible {
+  visibility: visible;
+  z-index: 2;
+}
+
+.region-dropdown ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a.region-dropdown-picker {
+  color: #505050;
+  font-size: .9rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+}
+
+a.region-dropdown-picker:after {
+  display: block;
+  margin-left: 15px;
+  color:rgba(20, 115, 230, 0);
+  transform-origin: center center;
+  transform: scaleX(2);
+  content: "\2228";
```

Comment:
This character looks like the same "down arrow" used to identify the popup in the region picker. I would recommend one that looks more like a checkmark.

Reply:
![Screen Shot 2021-02-15 at 1 41 52 PM](https://user-images.githubusercontent.com/62952234/107952227-c6262480-6f99-11eb-8b40-bd92bce48655.png)

Updated using CSS instead of `content` HTML checkmark